doc/manual/calling-c-and-fortran-code.rst

```diff
@@ -421,6 +425,8 @@ error if the Julia string contains any embedded NUL characters (which would caus
 truncated if the C routine treats NUL as the terminator).  If you are passing a ``char*`` to a C routine that
 does not assume NUL termination (e.g. because you pass an explicit string length), or if you know for certain that
 your Julia string does not contain NUL and want to skip the check, you can use ``Ptr{UInt8}`` as the argument type.
+``Cstring`` can also be used as the `ccall` return type, but that does not introduce extra checks and is only meant
```

Comment:
the antecedent of "that" in "that does not introduce extra checks" is unclear.  Better to write "`Ptr{UInt8}` does not introduce extra checks"